tests/verified.test.js

```diff
@@ -0,0 +1,63 @@
+import { describe, expect, test } from "@jest/globals";
+import { loadReviewsForMovie } from "../src/movies.js";
+import { apiAdapter } from "../src/reviewList.js";
+
+const API_BASE = "https://plankton-app-xhkom.ondigitalocean.app/api/";
+
```

Comment:
Should these tests not be made with mocked datasources?

Reply:
We already have unit tests with mocked data, these are integration/e2e tests and are more or less mostly to test the front-end code. So I wanted tests with real data to test if the site still works as it should. So it seamed unnecessary with mocked data for it?

Reply:
Cool! I wasn't sure what exactly the assignment said but if you have a good reason for skipping the mocking that's good enough for me. 😁